N°7077 Hungarian translations based on iTop 3.0.2-1

piRGoif opened the pull request:
This is an integration of @tacsaby translations
Many thanks for this big work !

I ran tests and fixed some placeholder errors
Then ran a manual check, some remarks will follow as PR comments

Also note that modifications are based on the support/3.0 branch, but product team might decide to integrate this work in another version (3.1.x or 3.2.0). This will be discussed during functional review.

Note that there is a corresponding PR for the webhook module : Combodo/combodo-webhook-integration#19

piRGoif committed:
Modifications from tacsaby

piRGoif committed:
Manual review

piRGoif committed:
Auto review
Based on tests results (DictionariesConsistencyTest, DictionariesConsistencyAfterSetupTest)

piRGoif commented:
I modified some "+" keys from the `'~~'` value to simply `''`
But there were lots of them so I stopped. 
In your future translations @tacsaby you may do this : it is a clear statement that the key was reviewed and stays with an empty value in this language.

piRGoif commented:
Alors, there were lots of "+" dict keys that remains with their english values, whereas they non "+" counterparts were translated. I would expect those "+" values would be emptied ?

What keys I saw : 

datamodels/2.x/itop-change-mgmt/dictionaries/hu.dict.itop-change-mgmt.php
Class:Change/Attribute:category/Value:application+
and the following keys


datamodels/2.x/itop-config-mgmt/dictionaries/hu.dict.itop-config-mgmt.php
Class:SoftwareInstance/Attribute:status/Value:active+
Class:SoftwareInstance/Attribute:status/Value:inactive+

Class:Model/Attribute:type/Value:Enclosure+
Class:Model/Attribute:type/Value:MobilePhone+
Class:Model/Attribute:type/Value:NAS+
Class:Model/Attribute:type/Value:NetworkDevice+
Class:Model/Attribute:type/Value:PC+
Class:Model/Attribute:type/Value:PDU+
Class:Model/Attribute:type/Value:Peripheral+
Class:Model/Attribute:type/Value:Printer+
Class:Model/Attribute:type/Value:Rack+
Class:Model/Attribute:type/Value:SANSwitch+


datamodels/2.x/itop-incident-mgmt-itil/dictionaries/hu.dict.itop-incident-mgmt-itil.php
Menu:IncidentManagement+

Class:Incident/Attribute:resolution_code/Value:assistance+
Class:Incident/Attribute:resolution_code/Value:bug fixed+
and the following keys

Class:Incident/Attribute:user_satisfaction/Value:1+
Class:Incident/Attribute:user_satisfaction/Value:2+
and the following keys


datamodels/2.x/itop-knownerror-mgmt/dictionaries/hu.dict.itop-knownerror-mgmt.php
Class:KnownError/Attribute:domain/Value:Application+
Class:KnownError/Attribute:domain/Value:Desktop+
Class:KnownError/Attribute:domain/Value:Network+
Class:KnownError/Attribute:domain/Value:Server+


datamodels/2.x/itop-request-mgmt/dictionaries/hu.dict.itop-request-mgmt.php
Class:UserRequest/Attribute:request_type/Value:incident+
Class:UserRequest/Attribute:request_type/Value:service_request+

Class:UserRequest/Attribute:priority/Value:1+
Class:UserRequest/Attribute:priority/Value:2+
and the following keys

Class:UserRequest/Attribute:urgency/Value:1+
Class:UserRequest/Attribute:urgency/Value:2+
and the following keys

Class:UserRequest/Attribute:origin/Value:mail+
Class:UserRequest/Attribute:origin/Value:monitoring+
Class:UserRequest/Attribute:origin/Value:phone+
Class:UserRequest/Attribute:origin/Value:portal+

Class:UserRequest/Attribute:escalation_flag/Value:no+
Class:UserRequest/Attribute:escalation_flag/Value:yes+

Class:UserRequest/Attribute:resolution_code/Value:assistance+
Class:UserRequest/Attribute:resolution_code/Value:bug fixed+
Class:UserRequest/Attribute:resolution_code/Value:hardware repair+
Class:UserRequest/Attribute:resolution_code/Value:other+
Class:UserRequest/Attribute:resolution_code/Value:software patch+
Class:UserRequest/Attribute:resolution_code/Value:system update+
Class:UserRequest/Attribute:resolution_code/Value:training+

Class:UserRequest/Attribute:user_satisfaction/Value:1+
Class:UserRequest/Attribute:user_satisfaction/Value:2+
Class:UserRequest/Attribute:user_satisfaction/Value:3+
Class:UserRequest/Attribute:user_satisfaction/Value:4+


datamodels/2.x/itop-request-mgmt-itil/dictionaries/hu.dict.itop-request-mgmt-itil.php
Menu:UserRequest:Provider+

Class:UserRequest/Attribute:request_type/Value:service_request+

Class:UserRequest/Attribute:priority/Value:1+
Class:UserRequest/Attribute:priority/Value:2+
and the following keys

Class:UserRequest/Attribute:urgency/Value:1+
Class:UserRequest/Attribute:urgency/Value:2+
and the following keys

Class:UserRequest/Attribute:origin/Value:mail+
Class:UserRequest/Attribute:origin/Value:monitoring+
Class:UserRequest/Attribute:origin/Value:phone+
Class:UserRequest/Attribute:origin/Value:portal+

Class:UserRequest/Attribute:escalation_flag/Value:no+
Class:UserRequest/Attribute:escalation_flag/Value:yes+

Class:UserRequest/Attribute:resolution_code/Value:assistance+
Class:UserRequest/Attribute:resolution_code/Value:bug fixed+
Class:UserRequest/Attribute:resolution_code/Value:hardware repair+
Class:UserRequest/Attribute:resolution_code/Value:other+
Class:UserRequest/Attribute:resolution_code/Value:software patch+
Class:UserRequest/Attribute:resolution_code/Value:system update+
Class:UserRequest/Attribute:resolution_code/Value:training+

Class:UserRequest/Attribute:user_satisfaction/Value:1+
Class:UserRequest/Attribute:user_satisfaction/Value:2+
and the following keys


datamodels/2.x/itop-service-mgmt/dictionaries/hu.dict.itop-service-mgmt.php
Class:SLT/Attribute:priority/Value:1+
Class:SLT/Attribute:priority/Value:2+
and the following keys

Class:SLT/Attribute:request_type/Value:incident+
Class:SLT/Attribute:request_type/Value:service_request+

Class:SLT/Attribute:unit/Value:hours+
Class:SLT/Attribute:unit/Value:minutes+


datamodels/2.x/itop-service-mgmt-provider/dictionaries/hu.dict.itop-service-mgmt-provider.php
Class:ServiceSubcategory/Attribute:request_type/Value:incident+
Class:ServiceSubcategory/Attribute:request_type/Value:service_request+

Class:SLT/Attribute:priority/Value:1+
Class:SLT/Attribute:priority/Value:2+
and the following keys

Class:SLT/Attribute:metric/Value:tto+
Class:SLT/Attribute:metric/Value:ttr+

Class:SLT/Attribute:unit/Value:hours+
Class:SLT/Attribute:unit/Value:minutes+


datamodels/2.x/itop-structure/dictionaries/hu.dict.itop-structure.php
Class:Contact/Attribute:notify/Value:no+
Class:Contact/Attribute:notify/Value:yes+

piRGoif commented:
I saw twice the same translation duplicated in the "+" keys, and same I would expect the "+" key to be empty instead of a duplicate of the label ?


datamodels/2.x/itop-request-mgmt-itil/dictionaries/hu.dict.itop-request-mgmt-itil.php
Menu:RequestManagementProvider+
Menu:UserRequest:EscalatedRequests+


dictionaries/hu.dictionary.itop.core.php
Core:AttributeEmailAddress+
Core:AttributeIPAddress+

piRGoif commented:
In the months names, 11 and 12 have the same value of the english labels, is it normal ?

dictionaries/hu.dictionary.itop.ui.php
Month-11
Month-12

piRGoif committed:
Revert Class:Contract/Attribute:cost_currency/Value:dollars keys changed to Forint

piRGoif commented:
datamodels/2.x/itop-service-mgmt-provider/dictionaries/hu.dict.itop-service-mgmt-provider.php
Class:Contract/Attribute:cost_currency/Value:dollars

DO NOT replace the dollar by your own currency. If you want to remove dollar and add your own national currency, create an iTop extension.

I did revert this change with d424a29e

piRGoif commented:
@tacsaby can you tell what you think of my different remarks ?

Hipska commented on datamodels/2.x/authent-cas/dictionaries/hu.dict.authent-cas.php:
```diff
@@ -6,7 +6,7 @@
+    'CAS:Error:UserNotAllowed' => 'Nem engedélyezett felhasználó',
+    'CAS:Login:SignIn' => 'Bejelentkezés CAS szerverrel',
+    'CAS:Login:SignInTooltip' => 'Kattintson ide az azonosításhoz a CAS szerveren',
+));
```
Missing newline

piRGoif replied:
Where ? Do you mean having an empty line on the end of file ?

Hipska replied:
Yes, see the [complete file](https://github.com/Combodo/iTop/pull/584/files/d424a29e3cf9a0cc474eea752cf4e254567347fa#diff-545c7b287c788f069e51d2ec5180c24df281c0028fa314002228119a2b360cd3), there is a missing EOL at EOF, which is shown by a ⛔ icon.

piRGoif replied:
Ok thanks.
This is not really a strict rule at Combodo.
I've done it this time (commit 3d372dde), but won't be blocking on next contributions.
Closing this conversation and other ones on same topic.

Hipska commented on datamodels/2.x/combodo-backoffice-darkmoon-theme/dictionaries/hu.dict.combodo-backoffice-darkmoon-theme.php:
```diff
@@ -22,5 +22,5 @@
 Dict::Add('HU HU', 'Hungarian', 'Magyar', array(
-	'theme:darkmoon' => 'Dark moon~~',
+	'theme:darkmoon' => 'Dark moon',
 ));
```
Missing newline

piRGoif replied:
Same question as discussion above ?

Hipska replied:
Same response as above.

Hipska commented on datamodels/2.x/itop-backup/dictionaries/hu.dict.itop-backup.php:
```diff
@@ -22,39 +22,39 @@
+    'bkp-wait-backup' => 'Várjon a mentés befejezéséig...',
+    'bkp-wait-restore' => 'Várjon a visszaállítás befejezéséig...',
+    'bkp-success-restore' => 'A visszaállítás sikerült.',
+));
```
Missing newline

Molkobain commented on datamodels/2.x/combodo-webhook-integration/hu.dict.combodo-webhook-integration.php:
This file is not part of this repo, it should be proposed on combodo-webhook-integration repo.

piRGoif replied:
File removed with 74ada97c
I've done a whole check again, didn't find any other concerned modules

piRGoif replied:
Modifications pushed to another PR in the module repo : Combodo/combodo-webhook-integration#19

Hipska commented on datamodels/2.x/itop-config/dictionaries/hu.dict.itop-config.php:
```diff
@@ -22,17 +22,17 @@
+    'config-parse-error' => '%2$d sor: %1$s.<br/>A fájl NEM frissült',
+    'config-current-line' => 'Szerkesztett sor: %1$s',
+    'config-saved-warning-db-password' => 'Sikeresen elmentve, de a biztonsági mentés nem fog működni az adatbázis jelszavában szereplő nem támogatott karakterek miatt.',
+));
```
Missing newline

Hipska commented on datamodels/2.x/itop-faq-light/dictionaries/hu.dict.itop-faq-light.php:
```diff
@@ -23,46 +23,47 @@
+    'Brick:Portal:FAQ:Title' => 'Tudástárak',
+    'Brick:Portal:FAQ:Title+' => '<p>Siet?</p><p>Nézze át a leggyakoribb kérdések listáját, és (talán) azonnal megtalálja a keresett választ.</p>',
 ));
+
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-attachments/dictionaries/hu.dict.itop-attachments.php:
```diff
@@ -21,67 +21,65 @@
+    'Class:Attachment/Attribute:user_id+' => '',
+    'Class:Attachment/Attribute:contact_id' => 'Kapcsolattartó',
+    'Class:Attachment/Attribute:contact_id+' => '',
+));
```
Missing newline

Hipska commented on datamodels/2.x/itop-change-mgmt-itil/dictionaries/hu.dict.itop-change-mgmt-itil.php:
```diff
@@ -61,232 +61,232 @@
+    'Class:Change/Attribute:outage/Value:no+' => '~~',
+    'Class:Change/Attribute:outage/Value:yes' => 'Igen',
+    'Class:Change/Attribute:outage/Value:yes+' => '~~',
+    'Class:Change/Attribute:fallback' => 'Visszavonás',
+    'Class:Change/Attribute:fallback+' => '~~',
+    'Class:Change/Attribute:parent_id' => 'Fölérendelt változás',
+    'Class:Change/Attribute:parent_id+' => '~~',
+    'Class:Change/Attribute:parent_name' => 'Referenciaszám',
+    'Class:Change/Attribute:parent_name+' => '~~',
```
Should empty `'~~'` always be removed? Maybe do a find and replace?

piRGoif replied:
May not be that bad, we'll see during reviews

Hipska replied:
Yeah, this PR is confusing as it is removing them on some places and in other places it is adding them, which is inconsistent.

piRGoif replied:
I agree ! 
We discussed this during the review : 
- no problem to keep `'~~'`
- for + keys having same value as their counterpart, we can keep exact duplicates as iTop handles it, but if the key is translated and not its + key, then either copy / paste the translation or empty the + key

piRGoif replied:
Reverted the '~~' + dict keys with 59233800
Closing conversation

Hipska commented on datamodels/2.x/itop-config-mgmt/dictionaries/hu.dict.itop-config-mgmt.php:
```diff
@@ -1512,66 +1510,66 @@
+    'Menu:Software' => 'Szoftverkatalógus',
+    'Menu:Software+' => 'Szoftverek',
 ));
 ?>
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-incident-mgmt-itil/dictionaries/hu.dict.itop-incident-mgmt-itil.php:
```diff
@@ -60,179 +60,180 @@
+    'Class:Incident/Method:ResolveChildTickets+' => 'A megoldás kaszkádozása a kapcsolódó jegyhez (ev_autoresolve), és a következő jellemzők összehangolása: szolgáltatás, csapat, ügyintéző, megoldási információ.',
+    'Tickets:Related:OpenIncidents' => 'Nyitott incidensek',
 ));
+
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-oauth-client/hu.dict.itop-oauth-client.php:
```diff
@@ -7,68 +7,68 @@
-	'Class:OAuthClientAzure/Name' => '%1$s (%2$s)~~',
+    'Class:OAuthClientAzure' => 'OAuth ügyfél Microsoft Azure-hoz',
+    'Class:OAuthClientAzure/Name' => '%1$s (%2$s)',
 
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-oauth-client/hu.dict.itop-oauth-client.php:
```diff
@@ -7,68 +7,68 @@
+    'Class:OAuthClientGoogle' => 'OAuth ügyfél a Google-höz',
+    'Class:OAuthClientGoogle/Name' => '%1$s (%2$s)',
 ]);
 
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-problem-mgmt/dictionaries/hu.dict.itop-problem-mgmt.php:
```diff
@@ -20,104 +20,104 @@
+    'UI-ProblemManagementOverview-ProblemUnassigned+' => '~~',
+    'UI:ProblemMgmtMenuOverview:Title' => 'Problémakezelés műszerfal',
+    'UI:ProblemMgmtMenuOverview:Title+' => '~~',
 
 ));
```
```suggestion
));

```

Hipska commented on datamodels/2.x/itop-request-mgmt/dictionaries/hu.dict.itop-request-mgmt.php:
```diff
@@ -55,229 +55,230 @@
+    'Organization:Overview:MyUserRequests' => 'Hozzám rendelt kérelmek ebből a szervezeti egységből',
+    'Organization:Overview:Tickets' => 'Hibajegyek ebből a szervezeti egységből',
 ));
+
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-service-mgmt-provider/dictionaries/hu.dict.itop-service-mgmt-provider.php:
```diff
@@ -96,441 +96,442 @@
+    'Class:lnkDeliveryModelToContact/Attribute:role_name' => 'Szerepkör név',
+    'Class:lnkDeliveryModelToContact/Attribute:role_name+' => '~~',
 ));
+
```
```suggestion
```

Hipska commented on datamodels/2.x/itop-service-mgmt/dictionaries/hu.dict.itop-service-mgmt.php:
```diff
@@ -65,417 +65,417 @@
+    'Class:lnkDeliveryModelToContact/Attribute:role_id+' => '~~',
+    'Class:lnkDeliveryModelToContact/Attribute:role_name' => 'Szerepkör név',
+    'Class:lnkDeliveryModelToContact/Attribute:role_name+' => '~~',
+));
```
Missing newline

Hipska commented on datamodels/2.x/itop-structure/dictionaries/hu.dict.itop-structure.php:
```diff
@@ -31,329 +31,330 @@
+    'theme:fullmoon' => 'Full moon',
+    'theme:test-red' => 'Tesztpéldány (Red)',
 ));
+
```
```suggestion
```

Hipska commented on dictionaries/ui/components/field/hu.dictionary.itop.field.php:
```diff
@@ -18,5 +18,5 @@
 Dict::Add('HU HU', 'Hungarian', 'Magyar', array(
-	'UI:Component:Field:BulkModify:UnknownValues:Tooltip' => 'Unknown values~~',
+	'UI:Component:Field:BulkModify:UnknownValues:Tooltip' => 'Ismeretlen értékek',
 ));
```
Missing newline

Hipska commented on dictionaries/ui/components/global-search/hu.dictionary.itop.global-search.php:
```diff
@@ -18,10 +18,10 @@
+    'UI:Component:GlobalSearch:LastQueries:NoQuery:Placeholder' => 'Még nem futtatott semmilyen keresést',
+    'UI:Component:GlobalSearch:HistoryDisabled' => 'Az előzmények le lettek tiltva',
+    'UI:Component:GlobalSearch:KeyboardShortcut:OpenDrawer' => 'Globális keresés megnyitása',
 ));
```
Missing newline

Hipska commented on dictionaries/ui/components/input/hu.dictionary.itop.input.php:
```diff
@@ -19,5 +19,5 @@
 Dict::Add('HU HU', 'Hungarian', 'Magyar', array(
-	'UI:Component:Input:Password:DoesNotMatch' => 'Passwords do not match~~',
+	'UI:Component:Input:Password:DoesNotMatch' => 'A jelszavak nem egyeznek',
 ));
```
Missing newline

Hipska commented on dictionaries/ui/layouts/navigation-menu/hu.dictionary.itop.navigation-menu.php:
```diff
@@ -18,17 +18,17 @@
+    'UI:Layout:NavigationMenu:UserInfo:Picture:AltText' => '%1$s névjegyképe',
+    'UI:Layout:NavigationMenu:UserMenu:Toggler:Label' => 'Felhasználói menü megnyitása',
+    'UI:Layout:NavigationMenu:KeyboardShortcut:FocusFilter' => 'Menüpontok szűrése',
 
```
```suggestion
```

Hipska commented on dictionaries/ui/pages/errorpage/hu.dictionary.itop.errorpage.php:
```diff
@@ -18,8 +18,8 @@
+    'UI:ErrorPage:UnstableVersion' => 'Ön egy instabil verziót használ, amely hibákat tartalmazhat. Ha úgy gondolja, hogy ez a hiba emiatt lépett fel, kérjük, küldje el visszajelzését!',
+    'UI:ErrorPage:KittyDisclaimer' => 'Egyetlen cica sem sérült meg ennek a GIF-nek a készítése és kiadása során.
 
 - The R&amp;D Team~~',
```
```suggestion
- The R&amp;D Team',
```

Hipska reviewed (changes requested):
Some cosmetic changes, I can't evaluate the correctness of the translations..

piRGoif committed:
Remove combodo-webhook-integration translation as it isn't part of the iTop repository
I will create another PR in this module repo

piRGoif committed:
itop-faq-light HU dict : remove one extra newline at EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-config-mgmt HU dict: remove closing PHP tag

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-incident-mgmt-itil HU dict: remove extra newline at EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-oauth-client HU dict: remove extra newline at EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-oauth-client HU dict: remove extra newline at EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-problem-mgmt HU dict: remove extra newline

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-request-mgmt HU dict: remove extra newline on EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-service-mgmt-provider HU dict: remove extra newline on EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
itop-structure HU dict: remove extra newline on EOF

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
dictionaries/ui/layouts/navigation-menu HU dict: remove extra newline

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

piRGoif committed:
dictionaries/ui/pages/errorpage/ HU dict: remove remaining ~~

Co-authored-by: Thomas Casteleyn <thomas.casteleyn@super-visions.com>

Molkobain commented:
Accepted during functional review:
  - Still corrections to be done
  - Ok for support/3.0 branch
  - If not ready before 3.0.4, will be merged in 3.0 branch anyway even though they will not be any 3.0.5 (upgrade to 3.1.x necessary)

Molkobain commented on datamodels/2.x/itop-config-mgmt/dictionaries/hu.dict.itop-config-mgmt.php:
`~~`added on some lines, should be reverted

piRGoif replied:
Should be fixed in all files now (wasn't easy :/)

Molkobain commented on datamodels/2.x/itop-problem-mgmt/dictionaries/hu.dict.itop-problem-mgmt.php:
Many `~~` added on empty entries

Molkobain commented on datamodels/2.x/itop-knownerror-mgmt/dictionaries/hu.dict.itop-knownerror-mgmt.php:
`~~` added on some empty entries

Molkobain commented on datamodels/2.x/itop-service-mgmt/dictionaries/hu.dict.itop-service-mgmt.php:
`~~` added on some empty entries

Molkobain commented on datamodels/2.x/itop-structure/dictionaries/hu.dict.itop-structure.php:
`~~` added on empty entries

Molkobain commented on datamodels/2.x/itop-tickets/dictionaries/hu.dict.itop-tickets.php:
`~~` added on empty entries

Molkobain commented on dictionaries/hu.dictionary.itop.core.php:
`~~` added on empty entries

Molkobain commented on dictionaries/hu.dictionary.itop.ui.php:
`~~` added on empty entries

piRGoif committed:
Add missing newline at EOF

piRGoif committed:
Revert "Manual review"

This reverts commit 258b9333f2d8494a7b6a9037147124d2128f25fc.

Hipska reviewed (approved):
Great work!

piRGoif committed:
Fix changing '' to '~~'
Some '~~' values were added back :(
Sometimes there are blocks where '' and '~~' were mixed... I replaced with '' nevertheless (too much work)

piRGoif committed:
Fix + keys to avoid non translated values
iTop will handle duplicates by non displaying the + key. If we have non translated value in the + key it will be displayed :/

piRGoif commented:
> Alors, there were lots of "+" dict keys that remains with their english values, whereas they non "+" counterparts were translated. I would expect those "+" values would be emptied ?

Should be fixed in 3b3d9c7f

piRGoif commented:
We are close to be ready to merge !

@tacsaby only one thing remaining, can you confirm the month names values ? 

dictionaries/hu.dictionary.itop.ui.php
Month-11
Month-12

Molkobain reviewed (approved):
Good work!

tacsaby commented:
Hello,

the same
11 is november
12 is december

Pierre Goiffon ***@***.***> ezt írta (időpont: 2023. dec.
15., P 17:44):

> We are close to be ready to merge !
>
> @tacsaby <https://github.com/tacsaby> only one thing remaining, can you
> confirm the month names values ?
>
> dictionaries/hu.dictionary.itop.ui.php
> Month-11
> Month-12
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Combodo/iTop/pull/584#issuecomment-1858168563>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AU2J7G6ZCGEFS25A7FKZZMDYJR45XAVCNFSM6AAAAABAQE4TGCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQNJYGE3DQNJWGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

piRGoif commented:
Many thanks for the answer @tacsaby !
We should be ready to merge, I'll check with the rest of the team ! (and also will add a README modification as it seems we forgot to add you in the contributors list last time :/ )

piRGoif committed:
Add tacsaby to the contributors list

Molkobain commented on README.md:
```diff
@@ -103,6 +103,7 @@ We would like to give a special thank you 🤗 to the people from the community
 - Seki, Shoji
 - Shilov, Vladimir
 - Stukalov, Ilya (a.k.a [@ilya](https://www.github.com/ilya)-stukalov)
+- Tarjányi, Csaba (a.k.a [tacsaby](https://github.com/tacsaby))
```
```suggestion
- Stukalov, Ilya (a.k.a [@ilya](https://www.github.com/ilya-stukalov))
- Tarjányi, Csaba (a.k.a [@tacsaby](https://github.com/tacsaby))
```

Molkobain committed:
Fix typo in README.md

piRGoif commented:
Hello,
I've merged the translations to the support/3.1 and develop branches. There were lots of conflicts on the support/3.1 merge... I've chose to keep the support/3.0 modifications on each modification.
@tacsaby can you please have a look ? You can comment directly in the corresponding merge commit (379062f), as a general commit or on specific lines
